setup.py

```diff
@@ -33,5 +33,6 @@
     package_data={'meneco': ['encodings/*.lp']},
     # scripts          = ['meneco.py'],
     entry_points={'console_scripts': ['meneco = meneco.__main__:main_meneco']},
+    tests_requires=['pytest'],

```

Comment:
This line is only appropriate when using `setup.py test`, which is deprecated. See also comment https://github.com/cfrioux/meneco/pull/9/commits/fba0163f8776f0d8fa7806260c7384eed44b8b15#r381012683